bootstrap/farmos_seed.py

```
@@ -0,0 +1,121 @@
+    print()
+    await print_summary()
+    await close_db()
+    print()
+    info("FarmOS 시드 완료")
+    return 0
+
+
+if __name__ == "__main__":
+    set_log_prefix(LOG_PREFIX)
+    try:
+        raise SystemExit(asyncio.run(run()))
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)